seed/data/dev/00650_workload_owner.js

```diff
@@ -0,0 +1,24 @@
+var tableName = 'workload_owner'
+
+exports.seed = function (knex, Promise) {
```

Comment:
Why not build this in to the other workload_owner seed file?

Reply:
was easier to write it in a new file, but I've now moved it into the original